docs/database/database-metrics.md

```diff
@@ -15,6 +15,8 @@ and attributes but more may be added in the future.
 
 <!-- toc -->
 
+- [Database call](#database-call)
+  * [Metric: `db.client.call.duration`](#metric-dbclientcallduration)
```

Comment:
since we use `db.operation` for attribute, I wonder if we should use `db.client.operation.duration` for consistency

Reply:
My original intuition was to use the term `operation` as well. The reason I tried `call` on for size is because the span conventions seem to use this term over operation.

I'll change this PR to use `operation`. I'm considering opening a separate PR to change the terminology in the span conventions to use `operation` so we can get a sense for what that looks like.